assets/js/googlesitekit/datastore/site/first-party-mode.js

```diff
@@ -80,6 +80,15 @@ const fetchSaveFirstPartyModeSettingsStore = createFetchStore( {
 	},
 } );
 
+const fetchGetFPMServerRequirementStatusStore = createFetchStore( {
```

Comment:
Separate test coverage for the fetch store action created by `createFetchStore` is not required, as the `createFetchStore` factory itself already covers it.

Reply:
Thanks @hussain-t. Makes sense, it's essentially just boilerplate and there's no logic to speak of in the reused reducer callback which is already covered elsewhere.